pkg/kubernetes/scripts/registry.js

```diff
@@ -163,6 +164,59 @@
                 return select().kind("Project");
             };
 
+            // TODO: invalidate cache on role changes
+            var sarCache = { };
+
+            $scope.subjectAccessReview = function subjectAccessReview(project, user, verb, resource) {
+                var result = false;
+
+                // if no project was given, check the first one as an approximation
+                // TODO: is that sufficient, or should we iterate over all
```

Comment:
This is find for the dashboard but other cases should probably always have a project.